pkg/apply/taskrunner/runner.go

```diff
@@ -258,6 +273,15 @@ func (b *baseRunner) run(ctx context.Context, taskQueue chan Task,
 	}
 }
 
+func handleWaitError(eventChannel chan event.Event, err error) {
```

Comment:
Why are we sending a new different event when continue on error is enabled?
If continue on error is disabled this would be an ErrorEvent, right?